src/enum.cr

```diff
@@ -373,6 +373,21 @@ struct Enum
     from_value?(value) || raise "Unknown enum #{self} value: #{value}"
   end
 
+  # Returns `true` if the given *value* is the enum member, or returns
+  # `false` if not member.
+  #
+  # ```
+  # Color.valid?(Color::Red)   # => true
+  # Color.valid?(Color.new(4)) # => false
+  # ```
```

Comment:
This example should be self-contained and include a definition of the used enum.

Reply:
Other enum methods (not touched by this issue) use `Color` in their examples. It is `Enum` docs issue.